.gitignore

```
@@ -1 +1,2 @@
 build
+*.parquet
```

We assume that our test `*.parquet`s exist in `mysql-test/suite/`, right?

```suggestion
/mysql-test/suite/*.parquet
```